### PR TITLE
Drools: add rhdm fixes and some formatting

### PR DESCRIPTION
**Pull request by tomasdavidorg**

### CODE REVIEW

**Comment by apodhrad** [comment]
@tomasdavidorg is the pr ready to be merged?

**Comment by tomasdavidorg** [comment]
@apodhrad yes

File: tests/org.jboss.tools.drools.ui.bot.test/resources/DslTestContent [hunk 1]
```diff
@@ -41,7 +41,7 @@
 [keyword]this is no a keyword=true
 [keyword][]true=false
 [keyword][java.lang.Object]false=true
-[conditions]příliš žluťoučký kůň úpěl ďábelské ódy=The quick brown fox jumps over the lazy dog
+[conditions]prilis zlutoucky kun upel dabelske ody=The quick brown fox jumps over the lazy dog
```
Comment (tomasdavidorg) [inline tests/org.jboss.tools.drools.ui.bot.test/resources/DslTestContent]: DROOLS-2356

**Comment by tomasdavidorg** [comment]
test this please

**Pull request merged**